apps/langchain/.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will run integration tests for the current project once per day
```

Comment:
Why do we want this instead of just running on commit/PR etc?

Reply:
This was actually auto-generated when initiating the langchain nx project! And we're safe to remove this as this won't work with nx monorepo anyway, if we do want to run tests on CI (which we don't have anyway and probably will never have for the langchain project), should be done from the monorepo.

And also yeah, weird cron job there!

aebe19d9392f73072f60526b2d1a37f0625fcc74